src/test/java/core/basesyntax/ProcessDataServiceTest.java

```diff
@@ -0,0 +1,66 @@
+package core.basesyntax;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.Fruit;
+import core.basesyntax.model.Operation;
+import core.basesyntax.service.ProcessDataService;
+import core.basesyntax.service.impl.ProcessDataServiceImpl;
+import core.basesyntax.strategy.BalanceOperationService;
+import core.basesyntax.strategy.OperationHandler;
+import core.basesyntax.strategy.OperationStrategy;
+import core.basesyntax.strategy.PurchaseOperationService;
+import core.basesyntax.strategy.ReturnOperationService;
+import core.basesyntax.strategy.SupplyOperationService;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class ProcessDataServiceTest {
+    private static ProcessDataService processDataService;
+
+    @BeforeClass
+    public static void initialSetup() {
+        processDataService = new ProcessDataServiceImpl();
+        Map<Operation, OperationHandler> operationHandler = new HashMap<>();
+        operationHandler.put(Operation.BALANCE, new BalanceOperationService());
+        operationHandler.put(Operation.SUPPLY, new SupplyOperationService());
+        operationHandler.put(Operation.PURCHASE, new PurchaseOperationService());
+        operationHandler.put(Operation.RETURN, new ReturnOperationService());
+        OperationStrategy.setOperationHandler(operationHandler);
+    }
+
+    @Test
+    public void addOnlyNewFruits_ok() {
+        List<Fruit> listToProcess = new ArrayList<>();
```

Comment:
perhaps in the name of the test it is necessary to add the name of a method which we test: "processData_addOnlyNewFruits_ok()"